### DISCUSSION TITLE
`ogma-core`: Type not filled in in ROS 2 app when using requirements DB

### DISCUSSION THREAD

**Discussion by ivanperez-keera** [nasa/ogma#538] [General]

**Comment by sylvesterkaczmarek** [comment]
Hi @ivanperez-keera, I'd like to work on this issue if it's available. I traced the failure to FRET requirements DB input leaving external variable types unset while the ROS variable database already contains the corresponding type information. I plan to backfill only missing Copilot types from the variable DB, preserve types already supplied by the specification, and add regression coverage. Please assign this issue to me if appropriate.

**Comment by ivanperez-keera** [comment]
Thanks, but not at this stage. This issue will most likely be closed.

**Comment by ivanperez-keera** [comment]
This will be abandoned for the foreseeable future. It if becomes a problem in the future, we can consider to open an issue.
